### ISSUE TITLE
Unable to Move Processed Files After run_kilosort Execution

### ISSUE THREAD

**Issue by Andrea3v** [MouseLand/Kilosort#829]
### Describe the issue:

Hi 

I am encountering a "PermissionError" when trying to move processed files (e.g., .raw data and kilosort.log) to a different drive after running 'run_kilosort'. 
The error suggests the files are still being accessed by kilosort process:
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process.

I suspect that the function does not release the file handles upon completion, preventing the files from being moved.
Is there a way to explicitly close/release file handles associated with the run_kilosort function once the processing is complete? Alternatively, could a feature be added to ensure files are properly released at the end of the execution? 

I reviewed the documentation at Kilosort API https://kilosort.readthedocs.io/en/latest/api.html#module-kilosort.run_kilosort but could not find a (obvious) solution.

thanks for any help or suggestion you may have
~Andre

### Version information:

kilosort.run_kilosort: Kilosort version 4.0.20
kilosort.run_kilosort: Python version 3.10.15

kilosort.run_kilosort: System information:
kilosort.run_kilosort: Windows-10-10.0.22631-SP0 AMD64
kilosort.run_kilosort: Intel64 Family 6 Model 183 Stepping 1, GenuineIntel
kilosort.run_kilosort: Using GPU for PyTorch computations. Specify `device` to change this.
kilosort.run_kilosort: Using CUDA device: NVIDIA GeForce RTX 3090 Ti 23.99GB

**Comment by j-moore94** [comment]
Hi, 
Just thought I'd add to this because I posted this problem last week and @jacobpennington said that it would be resolved in the new version:
https://github.com/MouseLand/Kilosort/issues/819#issue-2675621839

However I've just tried the new version (4.0.21) and I'm running into a new issue with the logging process. Now when I try to run more than one dataset in the same session using run_kilosort, I get the error: 

`FileNotFoundError: [Errno 2] No such file or directory: '[E:\\Data_for_kilosort\\3C303-S22\\kilosort4\\kilosort4.log]'`

i.e. it seems to be attempting to access the .log file from the previous dataset I ran. But it is not finding it because at the end my pipeline I copy the folder elsewhere and delete the local copy. Is there some way around this?

Thanks!

**Comment by Andrea3v** [comment]
Hi @jacobpennington thank you for your reply (and for your incredible work). Indeed Kilosort 4 ver. 4.0.21 works flawlessly - just tested it. 

hi @j-moore94 ,Thank you for your comment too. In the last run I did after updating, I looped through a bunch of recordings while setting the result_dir and used the` shutil.move()` command to move the preprocessed files to a different drive, which worked without any issues.

thanks again both
Andre
